impl/src/planning.rs

```diff
@@ -229,8 +230,8 @@ impl CrateCatalogEntry {
   pub fn local_build_path(&self, settings: &RazeSettings) -> String {
     match settings.genmode {
       GenMode::Remote => format!(
-        "remote/{}.{}",
-        &self.package_ident, settings.output_buildfile_suffix,
```

Comment:
We're removing this setting?

I'm OK with this, but maybe we should update the documentation of `output_buildfile_suffix` in documentation to mention that it doesn't do anything.

Reply:
It's still used in "Vendored" mode. My thought was update the variable. Your thoughts there? I do agree that something should be done though.

Reply:
I updated the documentation for that variable. I think it could honestly be deprecated though.